### ISSUE TITLE
Error on cluster without external snapshotter

### ISSUE THREAD

**Issue by NiklasRosenstein** [backube/volsync#929]
**Describe the bug**

I installed volsync on a Kubernetes cluster without external-snapshotter. According to [the documentation](https://volsync.readthedocs.io/en/stable/design/a-case-for.html)

> This project proposes to create a general method for supporting asynchronous, cross-cluster replication that can work with any storage system supporting a CSI-based storage driver. [...] For example, a simplistic CSI driver without snapshot capabilities should still be supported via a best-effort data copy [...]

But the volsync Pod fails with the following error (using chart version 0.7.1):

```
2023-10-09T23:41:53.217Z	ERROR	controller-runtime.source	if kind is a CRD, it should be installed before calling Start	{"kind": "VolumeSnapshot.snapshot.storage.k8s.io", "error": "no matches for kind \"VolumeSnapshot\" in version \"snapshot.storage.k8s.io/v1\""}
sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1.1
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/source/source.go:143
k8s.io/apimachinery/pkg/util/wait.runConditionWithCrashProtectionWithContext
	/go/pkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:235
k8s.io/apimachinery/pkg/util/wait.WaitForWithContext
	/go/pkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:662
k8s.io/apimachinery/pkg/util/wait.poll
	/go/pkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:596
k8s.io/apimachinery/pkg/util/wait.PollImmediateUntilWithContext
	/go/pkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:547
sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/source/source.go:136
```

**Steps to reproduce**

Install volsync on a cluster without the `VolumeSnapshot` CRD (k3s in my case).

**Additional context**

I want to run volsync on a k3s cluster that without `VolumeSnapshot` support.

**Comment by tesshuflower** [comment]
@NiklasRosenstein VolSync does depend on VolumeSnapshots for the majority of replications, so sets up watches on those resources (and therefore assumes you have the VolumeSnapshot CRD installed).  Are you planning on using VolSync in direct mode for your replications?

**Comment by NiklasRosenstein** [comment]
@tesshuflower Indeed, I was planning to use it in direct mode!

**Comment by onedr0p** [comment]
@NiklasRosenstein in the meantime you can install the snapshot controller and CRDs with [this helm chart](https://github.com/piraeusdatastore/helm-charts/tree/main/charts/snapshot-controller). The controller barely uses any resources in my cluster.

**Comment by tesshuflower** [comment]
Yes, sorry I should have mentioned - if you go with that workaround, and are not actually using snapshots, you shouldn't need the snapshot controller itself, only the CRDs.
